src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,16 @@
 import React from 'react';
+import { Comments } from '../../types/Comments';
+import { CommentInfo } from '../CommentInfo/CommentInfo';
+import './CommentList.scss';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  comments: Comments[];
+};
```

Comment:
Update the type of 'comments' to 'Comment[]' instead of 'Comments[]' to correctly represent an array of Comment objects.